sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -26,26 +26,54 @@ You can find the endpoint for your Azure AI Content Safety service resource usin
 az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
 ```
 
-#### Get the API key
+#### Create a ContentSafetyClient with API key
 
+To use an API key as the `credential` parameter.
+
+- Step 1: Get the API key. 
 The API key can be found in the [Azure Portal][azure_portal] or by running the following [Azure CLI][azure_cli_key_lookup] command:
 
-```bash
-az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
-```
+    ```bash
+    az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
+    ```
+
+- Step 2: Pass the key as a string into an instance of `AzureKeyCredential`.
 
-#### Create a ContentSafetyClient with AzureKeyCredential
+    ```python
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.contentsafety import ContentSafetyClient
+    
+    endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/"
+    credential = AzureKeyCredential("<api_key>")
+    client = ContentSafetyClient(endpoint, credential)
+    ```
 
-To use an API key as the `credential` parameter, pass the key as a string into an instance of `AzureKeyCredential`.
+#### Create a ContentSafetyClient with Service Principal
```

Comment:
It's better to not limited to Service Principal. Use the title "Create a ContentSafetyClient with Azure Active Directory credential"